201-servicebus-create-queue/azuredeploy.parameters.json

```diff
@@ -7,6 +7,12 @@
     },
     "serviceBusQueueName": {
       "value": "GEN-UNIQUE"
+    },
+    "serviceBusQueueName2": {
+      "value": "GEN-UNIQUE"
+    },
+    "serviceBusSku": {
```

Comment:
remove this and use the defaultValue in the template

Reply:
resolved